scope_shared/scope/schema.py

```diff
@@ -1,8 +1,9 @@
-import jschon
 import json
 from pathlib import Path
 from typing import Optional
 
+import jschon
```

Comment:
This auto-format change seems wrong to me, we'll use this to discuss the issue raised on Slack